doc/custom-build.md

```diff
@@ -66,32 +68,33 @@ Configuring pre-defined devices will allow the service to automatically provisio
 
 
 ### Configure the Device Service
-1. Open the [configuration.toml](./cmd/res/configuration.toml) file using your preferred text editor
+1. Open the [configuration.yaml](./cmd/res/configuration.yaml) file using your preferred text editor
 
-1. Make sure `secret name` is set to match `SecretName` in `camera.toml`. In the sample below, it is `"credentials001"`. If you have multiple cameras, make sure the secret names match.
+1. Make sure `secret name` is set to match `SecretName` in `camera.yaml`. In the sample below, it is `"credentials001"`. If you have multiple cameras, make sure the secret names match.
 
 1. Under `secretName`, set `username` and `password` to your camera credentials. If you have multiple cameras copy the `Writable.InsecureSecrets` section and edit to include the new information.
 
-```toml
-[Writable]
-    [Writable.InsecureSecrets.credentials001]
-    secretName = "credentials001"
-      [Writable.InsecureSecrets.credentials001.SecretData]
-      username = "<Credentials 1 username>"
-      password = "<Credentials 1 password>"
-      mode = "usernametoken" # assign "digest" | "usernametoken" | "both" | "none"
-
-    [Writable.InsecureSecrets.credentials002]
-    secretName = "credentials002"
-      [Writable.InsecureSecrets.credentials002.SecretData]
-      username = "<Credentials 1 password>"
-      password = "<Credentials 2 password>"
-      mode = "usernametoken" # assign "digest" | "usernametoken" | "both" | "none"
-
+```yaml
+Writable:
+  LogLevel: INFO
+  InsecureSecrets:
+    credentials001:
+      SecretName: credentials001
+      SecretData:
+        username: ""
+        password: ""
+        mode: usernametoken   # assign "digest" | "usernametoken" | "both" | "none"
+    # If having more than one camera, uncomment the following config settings
+    # credentials002:
+    #   SecretName: credentials002
+    #   SecretData:
+    #     username: ""
+    #     password: ""
+    #     mode: usernametoken    # assign "digest" | "usernametoken" | "both" | "none"
```

Comment:
Make YAML example match old TOML example. Don't simply copy what is in the configuration.yaml